tools/validators/instance_validator/validate/telemetry_error.py

```diff
@@ -39,8 +39,8 @@ def __eq__(self, other):
   def GetPrintableMessage(self):
     """Returns a human-readable message that explains this error."""
 
-    msg = "Error for entity [" + self.entity + "]"
-    if self.point:
-      msg += ", point [" + self.point + "]"
-    msg += ": " + self.message
+    msg = "- entity [" + self.entity + "]"
```

Comment:
nit: Maybe change the double quotes to single quotes to match our quote usage in other files? (My fault, I hadn't yet noticed the quote convention when I wrote this code.)